hiber/apps/bathouse/models.py

```diff
@@ -337,9 +337,9 @@ class HousePhysicalFeatures(models.Model):
     house_size = models.CharField(
         max_length=1,
         choices=(
-            ('S', 'Small'),
-            ('M', 'Medium'),
-            ('L', 'Large'),
+            ('S', 'Small - about a foot long'),
```

Comment:
This we'll address on the front end, and just say that it's a foot long, 1-3 feet long, etc.